src/roles/kserve/tests/create-devflag/verify.sh

```
@@ -0,0 +1,156 @@
+    error "Image SHA mismatch - Local: $local_sha, Remote: $remote_sha"
+    return 1
+  fi
+}
+
+verify_branch_pushed() {
+  local branch_name="${TEST_MANIFEST_BRANCH}"
+  local repo_dir=$1
+  local user_name=$2
+  local repo_name=$3
+  
+  cd $repo_dir
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)